poonam/JS/Assignment6_js/que5.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que5</title>
+        <script>
+            var input = [];
+            function inputnum() {
+                let num = true;
+                while(num) {
+                    let n = prompt('Enter a Number:');
+                    if (n != 'x') {
+                        input.push(parseInt(n));
+                    }
+                    else {
+                    num = false;
+                }
+                } 
+            }
+            function display() {
+                var newarray = [];
```

Comment:
long variable names should be like new_array like this so make this change in files where long variables are declared